anthropic-java-client-okhttp/src/main/kotlin/com/anthropic/client/okhttp/AnthropicOkHttpClient.kt

```diff
@@ -138,6 +146,17 @@ class AnthropicOkHttpClient private constructor() {
 
         fun authToken(authToken: Optional<String>) = authToken(authToken.orElse(null))
 
+        /**
+         * Sets the backend to be used for the backend service hosting Anthropic
+         * AI models. Implementations of the [Backend] interface can define the
+         * required credentials, prepare requests and handle responses for
+         * different backend services.
+         *
+         * @param backend The backend to be used. If connecting to the default
+         *     Anthropic backend service, a custom backend is not required.
+         */
+        fun backend(backend: Backend?) = apply { this.backend = backend }
```

Comment:
Can we add an `Optional<Backend>` version to match what we do for the other setters above